.github/workflows/test.yml

```diff
@@ -8,14 +8,18 @@ env:
 jobs:
   tests:
     name: "Tests"
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest]
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           persist-credentials: false
       - uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
-          python-version: 3.x
+          python-version: 3.12
```

Comment:
This value is "the Python version that's installed on the build server", I'm assuming?